docs/modules/mechanics/spawners.mdx

```diff
@@ -0,0 +1,204 @@
+---
+id: spawners
+title: Spawners
+---
+
+The <label>`<spawners>`</label> module provides an easy way of spawning entities and items into the world, without using actual spawner blocks and NBT editors.
```

Comment:
No need to add html `<label>` tags if a grave symbol (`) is already being used *and* it's not inside an html code block